.github/workflows/release.yml

```
@@ -0,0 +1,242 @@
+      
+      - name: Build binary
+        run: |
+          cargo build --release --target ${{ matrix.target }} --bin helium-server
+          cp target/${{ matrix.target }}/release/${{ matrix.artifact_name }} ${{ matrix.asset_name }}
+      
+      - name: Create tarball
+        run: |
+          tar czf ${{ matrix.asset_name }}.tar.gz ${{ matrix.asset_name }} README-COMETBFT.md LICENSE
+      
+      - name: Upload artifact
+        uses: actions/upload-artifact@v3
```

actionlint (1.7.7)

the runner of "actions/upload-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)